src/main/java/com/yurjinia/project_structure/ticket/service/mapper/TicketMapper.java

```diff
@@ -0,0 +1,60 @@
+package com.yurjinia.project_structure.ticket.service.mapper;
+
+import com.yurjinia.project_structure.comment.service.mapper.CommentMapper;
+import com.yurjinia.project_structure.ticket.dto.CreateTicketRequest;
+import com.yurjinia.project_structure.ticket.dto.TicketDTO;
+import com.yurjinia.project_structure.ticket.entity.TicketEntity;
+import com.yurjinia.user.dto.UserProfileDTO;
+import com.yurjinia.user.entity.UserEntity;
+import com.yurjinia.user.service.mapper.UserMapper;
+import com.yurjinia.user.service.mapper.UserProfileMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class TicketMapper {
+    private final UserMapper userMapper;
```

Comment:
all those mappers must not be here, if you need to map to dtos, use them on the higher level methods and provide dtos as arguments.